File: Lecture_code/L03S/student.js
function Student(units) {
  this.units = units;
  this.getUnits = getUnits;
  this.getStress = getStress;
}

function getUnits() {
	return this.units;
}

function getStress() {
	return 10*this.units;
}

patrick = new Student(16);
patrick.getStress();
getStress();

shayon = new Student(16);
shayon.name = "Shayon";
patrick.dog = "Molly";


function noStress() {
	return 0;
}

kasey = new Student(24);
kasey.getStress = noStress;

kasey.getStress();


dorm = {
    name: "FloMo",
	dog: "Molly the FloPup"
	};
	
dorm = {
    name: "FloMo",
	dog: "Molly the FloPup",
	printDogName: function() {
			document.writeln(this.dog);
		}
	};

File: Lecture_code/L03S/student.py
class Student:
	def __init__(self, units):
		self.units = units
		
	def getUnits(self):
		return self.units
		
	def getStress(self):
		return self.units * 10
		
patrick = Student(15)

patrick.getUnits()
patrick.getStress()

def setUnits(self, units):
        if (units < 0 or units > 20):
            return
        self.units = units

Student.setUnits = setUnits

patrick.setUnits(8)

setUnits(patrick,8)
